packages/renderer/src/lib/kube/pods/PodActions.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+import { faArrowsRotate, faExternalLinkSquareAlt, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { DropdownMenu } from '@podman-desktop/ui-svelte';
+import { createEventDispatcher, onMount } from 'svelte';
+
+import { withConfirmation } from '/@/lib/dialogs/messagebox-utils';
+
+import FlatMenu from '../../ui/FlatMenu.svelte';
+import ListItemButtonIcon from '../../ui/ListItemButtonIcon.svelte';
+import type { PodUI } from './PodUI';
+
+export let pod: PodUI;
+export let dropdownMenu = false;
+export let detailed = false;
+
+const dispatch = createEventDispatcher<{ update: PodUI }>();
+export let onUpdate: (update: PodUI) => void = update => {
+  dispatch('update', update);
+};
+
+$: openingKubernetesUrls = new Map();
```

Comment:
svelte5?

Reply:
I updated all the rest, but touching this one meant I had to switch to svelte5 events - and this one is currently propagating an event from the actions, through the svelte4 column on:update, through the svelte4 table on:update, back to the pods list...  and I got nervous. I'll try to do more testing; maybe svelte5 now has enough reactivity that the event can be safely removed.